docs/development/docs.rst

```diff
@@ -5,19 +5,10 @@ As one might expect,
 the documentation for Read the Docs is built using Sphinx and hosted on Read the Docs.
 The docs are kept in the ``docs/`` directory at the top of the source tree.
 
-You can build the docs by first ensuring this project is set up locally according to the :doc:`Installation Guide <install>`.
-Follow the instructions just up to the point of activating the virtual environment and then continue here.
-
-Next, install the documentation dependencies using ``pip`` (make sure you are inside of the virtual environment)::
-
-    pip install -r requirements/docs.txt
-
-This installs ``Sphinx``, amongst other things.
-
-After that is done, build the documentation by running::
-
-	# in the docs directory
-	make html
+.. TODO: expand this section explaining there the PR is automatically built and
+   the author can visualize changes without installing anything on their system.
+   However, if there is going to be periodic/bigger contributions, it may be a
+   good idea to install the Sphinx requirements to build our docs.
```

Comment:
We can't just delete the mention of Sphinx here? Isn't this just making our docs worse and more confusing for new contributors?

Reply:
IMO, I would just mention that people can just edit the files directly on GitHub and create a PR that automatically does the preview.

If there is going to be a bigger contributing at some time, we can write this guide at that point :smile:

Reply:
In fact, isn't it weird to have a "Contributing to Documentation" document under "Developer documentation" if we want to avoid referencing local installation instructions here? Perhaps we should move this to some other section, expand the content guidelines, and refer to `development/standards` for further information. But definitely not in this cleanup PR?

Reply:
@astrojuanlu I agree. In fact, do have a "Contributing to Read the Docs" (https://docs.readthedocs.io/en/stable/contribute.html) and we have a section for "Contributing to Documentation". Probably improving that section is a cleaner path to move forward.

Reply:
Yea, I think it's a bit chicken & egg, since we want it to live with "contributing", which mostly makes sense under dev docs. Not sure the best way to structure this.